main.cpp

```diff
@@ -0,0 +1,96 @@
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/opencv.hpp>
+#include "CommandLineParser.h"
+
+
+
+struct position
+{
+    std::vector<int> bestRow {};
```

Comment:
nit: there is default constructor for vector so no need for {} initialization
```suggestion
    std::vector<int> BestRow;
```